workspaces/src/error/mod.rs

```diff
@@ -38,15 +38,17 @@ enum ErrorRepr {
         kind: ErrorKind,
         message: Cow<'static, str>,
     },
-    #[error("{error}")]
+    #[error("{kind}")]
     Custom {
         kind: ErrorKind,
+        #[source]
         error: Box<dyn std::error::Error + Send + Sync>,
     },
-    #[error("{message}: {error}")]
+    #[error("{message}")]
```

Comment:
Curious in these why for `Custom` the error is not included in this message (does source automatically include it?), and for `Full`, why the error kind is ignored

Reply:
yup, `#[source]` creates a chain of error message when you go to display it. For `full`, didn't see a need to include it since the error message should for the most part tell you what kind of class of errors it it. Normally I don't think `kind` is that useful in error messages unless there's no other context to the error